websocket_server.cpp

```diff
@@ -10,6 +10,8 @@
 
 #ifdef USE_WEBSOCKETS
 
+#include "de_web_plugin.h"
+#include "de_web_plugin_private.h"
```

Comment:
Are these included for `deCONZ::appArgumentString()`?

`#include "deconz/util.h"`  should do the trick :)

Reply:
Indeed. Seems to work 👍